tox.ini

```diff
@@ -26,7 +26,6 @@ envlist =
 
     ; opentelemetry-example-app
     py3{5,6,7,8}-test-core-example-app
-    pypy3-test-core-example-app
```

Comment:
Please not the removal of tests for the Example App on pypy3. The Example App uses `grpcio` which is not available on `pypy` [packages](http://packages.pypy.org/).